src/student.rb

```diff
@@ -1,8 +1,8 @@
-require 'person'
+require './person'
```

Comment:
- [ ] The correct way to invoke a file in the same directory is using `require_relative` instead of `require` it is used to load modules or libraries that are globally installed on the system or are available in Ruby's module search path. Please 🙏🏽 use `require_relative` instead of `require` to call the app class.